Pipfile

```diff
@@ -12,20 +12,20 @@ markers = "python_version < '3.0'"
 black = '*'
 invoke = '*'
 parver = '*'
-passa = { editable = true, path = '.', extras = ['tests'] }
+passa = {editable = true,path = '.',extras = ['tests']}
 sphinx = '*'
 sphinx-rtd-theme = '*'
 towncrier = '*'
 twine = '*'
 wheel = '*'
+coverage = "<5.0"
```

Comment:
Might want a comment explaining why 5.0+ does not work.

Reply:
Not sure it doesn’t but I’ve had a lot of bad experiences leaving this unpinned lately as 5.x is still in alpha and doesn’t tend to play nice with xdist on Travis due to using SQLite as a shared backing store